Makefile

```diff
@@ -49,6 +49,7 @@ all: lint doc test integration_test demo
 # <regex>.
 lint: $(HEADERS) $(SOURCES)
 	cpplint --exclude=tools/docgen_main.cc \
+            --exclude=glad/src/glad.cc \
```

Comment:
This can be removed after you move the glad files.

Reply:
This doesn't seem to work. To compile `glad.cc` I have to include `external/glad/src` to `DIRS`. But then `cpplint` tries to check `glad.cc`, which is undesired.